app/models/direct_award.py

```diff
@@ -23,6 +24,7 @@ def serialize(self, with_users=False):
             "id": self.id,
             "name": self.name,
             "createdAt": self.created_at.strftime(DATETIME_FORMAT),
+            "downloadedAt": self.downloaded_at.strftime(DATETIME_FORMAT) if self.downloaded_at is not None else None,
```

Comment:
We could have a function in dmutils that knows how to handle the null case, if this gets very tedious, perhaps.